bin/node/runtime/src/lib.rs

```diff
@@ -128,9 +128,21 @@ impl frame_system::Trait for Runtime {
 	type ModuleToIndex = ();
 }
 
+parameter_types! {
+	// One storage item; value is size 4+4+8+32 bytes = 48 bytes.
```

Comment:
I assume this is in the following order: height, index, deposit, depositor?
But isn't deposit is of `Balance` type which is defined as `u128`?

```suggestion
	// One storage item; value is size 4+4+16+32 bytes = 56 bytes.
```